sparkprs/models.py

```diff
@@ -189,6 +193,18 @@ def _compute_commenters(self):
                         Issue.ASKED_TO_CLOSE_REGEX.search(comment['body']) is not None)
         return sorted(res.items(), key=lambda x: x[1]['date'], reverse=True)
 
+    def refresh(self):
```

Comment:
I wrote my earlier comment without the code handy. What I _meant_ to suggest was that the `clean_deleted_prs()` cron task just enqueue runs of the `update_pr` task (with `taskqueue.add(url=url_for(".update_pr", pr_number=pr['number'])`) and just modify `update_pr` to add a `try-except` block around the first `raw_github_request` call in that method.